src/redis.ts

```diff
@@ -60,7 +60,10 @@ export const setRedisObjectWithExpiry = (
   key: string,
   value: RedisObject,
   seconds: number,
-) => ioRedisPool.execute((client) => client.set(key, value, 'EX', seconds));
+) =>
+  ioRedisPool.execute((client) =>
+    client.set(key, value, 'EX', Math.floor(seconds)),
```

Comment:
We had this multiple times, so I think it is in fact better to fix in this level. Redis accepts only int (seconds) as expires.